kovri.i2p/_i18n/fr/contributing.md

```diff
@@ -1,72 +1,73 @@
-   - Commit messages should be verbose by default, consisting of a short subject line (50 chars max), a blank line, and detailed explanatory text as separate paragraph(s) - unless the title alone is self-explanatory.
-   - Commit title should prepend class or aspect of project. For example, "HTTPProxy: implement User-Agent scrubber. Fixes #193." or "Garlic: fix uninitialized padding in ElGamalBlock".
-   - If a particular commit references another issue, please add a reference. For example "See #123", or "Fixes #123". This will help us resolve tickets when we merge into ```master```.
-   - In general, commits should be [atomic](https://en.wikipedia.org/wiki/Atomic_commit#Atomic_commit_convention) and diffs should be easy to read. For this reason, please try to not mix formatting fixes with non-formatting commits.
-   - The body of the pull request should contain an accurate description of what the patch does and provide justification/reasoning for the patch (when appropriate). You should include references to any discussions such as other tickets or chats on IRC.
+## Conformité
+- Nous ésperons compléter la conformité C++11/14; n'hésitez pas à l'utiliser pour vous faciliter la tâche dans votre travail
+- Il est aussi vivement recommandé d'utiliser la librairie standard et les bibliothèques de dépendances si vous le pouvez
+
+## Envoyer votre travail
+Pour mettre à contribution votre travail, veuillez procéder de la façon suivante:
 
-## Proposals
-To contribute a proposal, please review our [open issues](https://github.com/monero-project/kovri/issues) for existing proposals. If what you propose is not there, then [open a new issue](https://github.com/monero-project/kovri/issues/new).
+1. Fork Kovri
+2. Lisez notre [style guide](https://github.com/monero-project/kovri/blob/master/doc/STYLE.md)
+3. Creez une [branche pour le sujet concerné](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
+4. [**Signez**](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work) votre ou vos commit(s)
+5. Envoyez une pull-request à la branche ```master```
+   - Nous n'avons pas encore de tags du au fait que nous soyons en pré-alpha. Donc pour l'instant vous pouvez directement travailler dans la branche master.
+   - Les messages de commmit doivent être détaillés par défaut, accompagné d'un message court d'environ 50 caractères à propos du sujet, une ligne blanche, et d'un texte détaillé expliquatoire constitué d'un ou plusieurs paragraphe(s) séparés - à moins que le titre explique à lui même le commit.
+   - Le titre du commit doit préciser la class ou l'aspect du projet concerné. Par exemple, "HTTPProxy: implement User-Agent scrubber. Fixes #193." ou "Garlic: fix uninitialized padding in ElGamalBlock".
+   - Si un commit particulier se réfère à un autre problème, veuillez dans ce cas ajouter une réference. Par exemple "Voir #123", ou "#123 résolu". Cela nous aidera à résoudre les tickets quand nous mergerons dans ```master```.
```

Comment:
We may need to specify that everything must me commented/typed in English, we may end up with multiple language PR and will be hard to handle.